Cargo.toml

```diff
@@ -13,9 +13,22 @@ readme = "./README.md"
 [dependencies]
 trustfall = "0.7.1"
 rustdoc-types = "0.29.1"
+rayon = { version = "1.10.0", optional = true, default-features = false }
+
+[features]
+default = ["rayon"]
+rayon = ["dep:rayon"]
+
+[[bench]]
+name = "indexed_crate"
+harness = false
+
+[profile.bench]
+debug = true
```

Comment:
This feels like a very aggressive debug setting to enable by default in `bench` runs:
https://doc.rust-lang.org/cargo/reference/profiles.html#debug

Is this considered good practice for `bench` targets in other Rust projects? Would a less intensive version be sufficient for our perf-testing needs?

Reply:
`cargo-flamegraph` is very pedantic about this so I added it to shut it up, I'll try with different values though!

Reply:
Ah TIL, that's also useful context to include. For example, `cargo-flamegraph` may change its recommendation in the future, and we want to know that this value is intended to match what `cargo-flamegraph` recommends and not due to some other thing. That will help us change the value appropriately in the future, without concerns about breaking something we don't understand!

Reply:
AFAIK `cargo-flamegraph is being too agressive with its request, `debug = 1` seems to provide the same amount of information.